lib/src/io/node.dart

```diff
@@ -187,32 +223,44 @@ T _systemErrorToFileSystemException<T>(T callback()) {
   }
 }
 
-final stderr = Stderr(process.stderr);
+Stderr get stderr {
+  var stderr_ = process?.stderr;
+  if (stderr_ == null) {
+    throw UnsupportedError("stderr is only supported on Node.js");
+  }
+  return Stderr(stderr_);
+}
 
 /// We can't use [process.stdout.isTTY] from `node_interop` because of
 /// pulyaevskiy/node-interop#93: it declares `isTTY` as always non-nullably
 /// available, but in practice it's undefined if stdout isn't a TTY.
-@JS('process.stdout.isTTY')
-external bool? get isTTY;
+// ignore: invalid_null_aware_operator
+bool get hasTerminal => process?.stdout?.isTTY == true;
 
-bool get hasTerminal => isTTY == true;
+bool get isWindows => process?.platform == 'win32';
 
-bool get isWindows => process.platform == 'win32';
+bool get isMacOS => process?.platform == 'darwin';
 
-bool get isMacOS => process.platform == 'darwin';
+bool get isJS => true;
 
-bool get isNode => true;
+bool get isNode => process != null;
 
 // Node seems to support ANSI escapes on all terminals.
 bool get supportsAnsiEscapes => hasTerminal;
 
-String get currentPath => process.cwd();
+String get currentPath => process?.cwd() ?? '/';
```

Comment:
This looks unused; you can delete it from here and the other two IO files.

Reply:
Removed in https://github.com/oddbird/dart-sass/commit/1937affd6fa9fc3965c72222f323f56fb53352d7